docker/context/config.py

```diff
@@ -0,0 +1,38 @@
+import os
+import json
+import hashlib
+from .. import utils
+
+CONTEXTS_DIR = os.path.join(os.path.dirname(
+    utils.find_docker_config_file() or ""), "contexts")
+METADATA_DIR = os.path.join(CONTEXTS_DIR, "meta")
+TLS_DIR = os.path.join(CONTEXTS_DIR, "tls")
+METAFILE = "meta.json"
+
+
+def get_current_context_name():
```

Comment:
Move to `Context` ?

Reply:
I prefer to leave it there as it just reads a field from the docker config.